book/tutorials/git_and_jupyterlab/jupyter.md

```diff
@@ -185,7 +192,7 @@ just went with a more descriptive label instead of the default value.
 # Change to the website folder, it was created with the `clone`
 cd website
 # Add the remote
-git remote add Hackweek {{ '`{url}.git`'.format(url=website_url) }}
+git remote add Hackweek https://github.com/snowex-hackweek/website2022.git
```

Comment:
```suggestion
git remote add Hackweek {{ '{url}.git'.format(url=website_url) }}
```

Reply:
See the above discussion:
https://github.com/uwhackweek/jupyterbook-template/pull/113#discussion_r965595340